src/Http/Http.Results/test/ResultsOfTHelperTests.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http.Metadata;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.DotNet.RemoteExecutor;
+
+namespace Microsoft.AspNetCore.Http.HttpResults;
+
+public class ResultsOfTHelperTests
+{
+    [ConditionalTheory]
+    [RemoteExecutionSupported]
```

Comment:
I missed this in @brunolins16's PR

The way we do this in dotnet/runtime is:

```C#
[ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
```

Do we need 2 separate attributes? Are we using the same `ConditionalTheory` attribute here as in dotnet/runtime?

Reply:
We use separate attributes everywhere else in ASP.NET Core. IMO lets keep the current pattern.

Reply:
Sounds good to me.